docs/src/pages/_document.page.tsx

```diff
@@ -25,7 +25,7 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       style-src 'self' 'unsafe-inline';
       font-src 'self' data:;
       frame-src 'self' *.codesandbox.io;
-      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net;
+      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net images.unsplash.com;
```

Comment:
For the example apps, I'm using public domain images from Unsplash. However, the docs site CSP rules were blocking the images from getting loaded. 

First, I experimented with adding a separate `/assets` folder and putting the images in there. CSP was still blocking the images for some reason, but I'm sure we could make that approach work if we needed to. 

The main reason I like the approach of directly passing the image URL's to the `Image` component's `src` attribute is because it allows the customer to copy/paste just one file to get the example app working. It would be a cumbersome experience to guide the customer in creating an `/assets` folder in their project and then copying all the images to that folder. 

However, if this presents any kind of security concerns, then I'll find a workaround.